modules/nw-ovn-kubernetes-migration-about.adoc

```diff
@@ -44,8 +44,18 @@ While the OVN-Kubernetes network plugin implements many of the capabilities pres
 
 * OVN-Kubernetes, the default network provider in {product-title} 4.14 and later versions, uses the following IP address ranges internally: `100.64.0.0/16`, `169.254.169.0/29`, `100.88.0.0/16`, `fd98::/64`, `fd69::/125`, and `fd97::/64`. If your cluster uses OVN-Kubernetes, do not include any of these IP address ranges in any other CIDR definitions in your cluster or infrastructure.
 
+* The OpenShift SDN plugin allows application of the `NodeNetworkConfigurationPolicy` (NNCP) custom resource (CR) to the primary interface on a node. The OVN-Kubernetes network plugin does not have this capability.
```

Comment:
In 4.16 you have this after the sentence on line 49. This is also duplicated on line 55. https://github.com/openshift/openshift-docs/pull/80277/files#diff-cb846332fe09a27629a19fa569e0acdca3d3595c4f4d423806868d06a89c9959L61. I _think_ this line just needs to be deleted.

Reply:
Valid point and well spotted. I removed this duplicate paragraph.